src/components/gameScreen/gameScreen.component.js

```diff
@@ -1,75 +1,68 @@
 import React from 'react';
 import {Text, View, StyleSheet, FlatList} from 'react-native';
 import Animated from 'react-native-reanimated';
+import { observer } from "mobx-react-lite";
 import {CustomButton} from './components/customButton';
+import {store} from '../../store/store';
+import { bookmarkStore } from "../../store/bookmarkStore";
 
-export const GameScreenComponent = ({
+const questionIcon = require('../../assets/img/icons/questionIcon.png');
+const bookmarkIcon = require('../../assets/img/icons/bookmarkIcon.png');
+
+export const GameScreenComponent = observer(({
   questionNumber,
   numberOfQuestions,
-  currentQuestion,
   answersAnimation,
   renderAnswerItem,
-  chosenMode,
   timer,
   nextButton,
   timerless,
-  mainColor,
-  questionIcon,
-  bookmarkIcon,
-  showHelpDialog,
   bookmarkSetter,
   renderHelpDialog,
   onOpenHelpDialog,
-  answers,
-  isBookmarkSet,
-  renderBookmarkStatus,
-  setIsButtonPressed,
-  isButtonPressed
+  renderBookmarkStatus
 }) => {
   return (
     <View style={styles.container}>
-      {showHelpDialog && renderHelpDialog()}
-      {isButtonPressed && renderBookmarkStatus()}
+      {store.showHelpDialog && renderHelpDialog()}
+      {bookmarkStore.isButtonPressed ? renderBookmarkStatus() : <></>}
```

Comment:
if isButtonPressed a boolean value then you should write it like:
`bookmarkStore.isButtonPressed && renderBookmarkStatus()`